### DISCUSSION TITLE
Syncing deleted notes

### DISCUSSION THREAD

**Discussion by boydkelly** [ohare93/brain-brew#30] [Ideas]
One issue that it seems crowd anki and git have not been able to solve is dealing with deleted notes.   Either at the Anki source or with something (like brainbrew) that syncs to git.   In my previous life with awk, and google docs, I was tagging notes, and then manually deleting on both sides.   

Or alternatively within Anki instead of deleting notes I would move them to a special <pending> deck.  Then that deck would also be published within gitlab pages, and I had a google apps script that used a function to import that and then deleted those in my main file.  That main file of course also published via google and downloaded with wget and merged back.   I already need an advil....

Anyway I wish that crowdanki would have an option to delete records in anki that are *not* in the imported file.

Does anyone have any other workflow to deal with this?

**Comment by jeprecated** [comment]
That is certainly something I'd like to get working! And yes, it would be best to do it on the CrowdAnki side. However I have a potential way to allow Brain Brew to inform CrowdAnki which exact cards/notes to delete 👏 Then the user would be prompted for this in the import config window. No timeline for this, though.
